debug_scripts/Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 boto3 = "==1.21.34"
-requests = "==2.27.1"
+requests = "==2.31.0"
```

Comment:
`debug_scripts/Pipfile`, `debug_scripts/Pipfile.lock` and `tools/debug-ui/package-lock.json` are not my changes. They may disappear once I sync with master. Meanwhile please ignore these files.